test/spec/libraries/consul_config_v1_spec.rb

```diff
@@ -320,6 +320,38 @@
         end
       end
 
+      describe 'connect_not_supported' do
+        before do
+          recipe = double('Chef::Recipe')
+          allow_any_instance_of(Chef::RunContext).to receive(:include_recipe).and_return([recipe])
+          default_attributes['consul']['version'] = '1.1.0'
+        end
+        recipe do
+          consul_config '/etc/consul/default.json' do
+            connect(enabled: true)
+          end
+        end
+        it 'does not set the [`discovery_max_stale`] field since unsupported' do
+          expect(config['connect']).to be_nil
+        end
+      end
+
+      describe 'connect_supported' do
+        before do
+          recipe = double('Chef::Recipe')
+          allow_any_instance_of(Chef::RunContext).to receive(:include_recipe).and_return([recipe])
+          default_attributes['consul']['version'] = '1.2.0'
+        end
+        recipe do
+          consul_config '/etc/consul/default.json' do
+            connect(enabled: true)
+          end
+        end
+        it 'sets the [`discovery_max_stale`] field' do
```

Comment:
Again, should be `connect` instead of `discovery_max_stale`?

Reply:
DONE